copier/profiles.yml

```diff
@@ -0,0 +1,10 @@
+---
+# profile selection
+template_profile:
+  type: str
+  default: ask
+  help: Select a profile
+  choices:
+    Recommended (our defaults for research software): "recommended"
```

Comment:
Error `inja2.exceptions.UndefinedError: 'full_name' is undefined` when 
Select a profile --> Recommended
Do you want to add extra details for the package? --->n

Reply:
I think this happens due to some broken design logic. I am happy to leave it as an issue and address later.